src/query/interpreter.cpp

```diff
@@ -1909,6 +1918,119 @@ PreparedQuery PrepareSettingQuery(ParsedQuery parsed_query, const bool in_explic
   // NOLINTNEXTLINE(clang-analyzer-cplusplus.NewDeleteLeaks)
 }
 
+Callback HandleTransactionQueueQuery(TransactionQueueQuery *transaction_query,
```

Comment:
can we create a TransactionQueueHandler to offload some of the lines from Callback to the object?

Reply:
see ReplQueryHandler

Reply:
shouldn't be a problem